tests/base/tensor_accessors_02.cc

```diff
@@ -76,8 +76,10 @@ int main()
 
     // via std::array
 
+    // via std::array:
 #ifdef DEAL_II_WITH_CXX11
-    deallog << TensorAccessors::extract<5>(foo, std::array<unsigned int, 5>({2, 1, 0, 2, 1})) << std::endl;
```

Comment:
On Mon, Sep 14, 2015, at 11:01 CDT, Wolfgang Bangerth <notifications@github.com> wrote:

>>  #ifdef DEAL_II_WITH_CXX11
>> - deallog << TensorAccessors::extract<5>(foo, std::array<unsigned
>> int, 5>({2, 1, 0, 2, 1})) << std::endl;
>
> The correct C++11 syntax would have been
> ```
>   std::array<...>{2,1,0,2,1}
> ```
> I believe, i.e., without the round parentheses.

Ah, yes. But with parentheses is also valid syntax. I've just used the
same syntax I know that gcc-4.6 understands.